Components/Spinner/README.md

```diff
@@ -0,0 +1,10 @@
+# Spinner
```

Comment:
Does storybook use this file somehow? If not, it would probably be better as a comment header in the ` Components/Spinner/index.jsx` file.

Reply:
Ignore this, I see the code now.